docs/quickstart.ipynb

```diff
@@ -6,45 +6,141 @@
    "source": [
     "# Getting Started with PyFixest\n",
     "\n",
-    "In a first step, we load the module and some example data:"
+    "## What is a fix effect model?\n",
+    "\n",
+    "A fixed effect model is a statistical model that includes fixed effects, which are parameters that are estimated to be constant across different groups. In the context of panel data, fixed effects are parameters that are constant across different individuals or time.\n",
```

Comment:
Would it make sense to introduce some panel data regression notation here? Or would that overly complicated things? E.g. formalise a regression equation of the type 

$$
Y_it = X_it \beta + alpha_i + \psi_t + \epsilon_it
$$

where we then explain that alpha_i is a unit fixed effect, psi_t is a time fixed effect, and mention that the unit fixed effect controls for all time invariant observed or unobserved characteristics of individual i, etc? 

Or would that be too much content already?

Also, should we mention that fixed effects are commonly used in panel setting, but that one does not need a panel data set to work with fixed effects? E.g. cluster randomized trials with cluster fixed effects, or wage regressions with worker and firm fixed effects?

Reply:
Added some more explanation as suggested in https://github.com/s3alfisc/pyfixest/pull/436/commits/eadcbf892160111010b44a6e8642ca220d690322